app/src/main/java/org/openimis/imispolicies/Enquire.java

```diff
@@ -393,8 +393,8 @@ public void run() {
                              TotalDelivieriesLeft = jsonObject.getString("totalDelivieriesLeft").equalsIgnoreCase("null")? "": "TotalDelivieriesLeft: "+jsonObject.getString("totalDelivieriesLeft");
                              TotalAntenatalLeft = jsonObject.getString("totalAntenatalLeft").equalsIgnoreCase("null")? "": "TotalAntenatalLeft: "+jsonObject.getString("totalAntenatalLeft");
                              ConsultationAmountLeft = jsonObject.getString("consultationAmountLeft").equalsIgnoreCase("null")? "": "ConsultationAmountLeft: "+jsonObject.getString("consultationAmountLeft");
```

Comment:
"ConsultationAmountLeft: " is hardcoded. Please add this in strings.xml.

Reply:
Fixed